client/src/pages/Message/SideBar/Content/Content.tsx

```diff
@@ -0,0 +1,25 @@
+import { Box, Typography } from '@material-ui/core';
+import { parseISO } from 'date-fns';
+import { format } from 'path';
+import React from 'react';
+import { Conversation } from '../../../../interface/Message';
+import useStyles from './useStyles';
+
+export interface ConversationListProps {
+  conversationItem: Conversation;
+}
+
+export default function Content({ conversationItem }: ConversationListProps): JSX.Element {
+  const classes = useStyles();
+  return (
+    <Box className={classes.root}>
```

Comment:
Same as comment above for MUI Box components

Reply:
added CSS property inside the Box component.